lib/pure/os.nim

```diff
@@ -944,17 +944,18 @@ proc rawRemoveDir(dir: string) =
   else:
     if rmdir(dir) != 0'i32 and errno != ENOENT: raiseOSError(osLastError())
 
-proc removeDir*(dir: string) {.rtl, extern: "nos$1", tags: [
+proc removeDir*(dir: string, recursive = true) {.raises: [OSError], rtl, extern: "nos$1", tags: [
   WriteDirEffect, ReadDirEffect], benign.} =
-  ## Removes the directory `dir` including all subdirectories and files
-  ## in `dir` (recursively).
+  ## Removes the directory ``dir``. Additionally removes all subdirectories and files recursively
+  ## if ``recursive`` is ``true`` (default behavior is recursive).
```

Comment:
```diff
-  ## if ``recursive`` is ``true`` (default behavior is recursive).
+  ## if ``recursive`` is ``true``.
```

I personally think that the emphasis is not necessary, since default values are shown in the doc already (eg. [walkDir](https://nim-lang.org/docs/os.html#walkDir.i%2Cstring))

Reply:
Agreed.